src/app/view/components/dot-form-dialog/dot-form-dialog.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="dot-form-dialog__content">
-    <ng-content></ng-content>
+    <ng-content pFocusTrap></ng-content>
```

Comment:
what is `pFocusTrap ` for?

Reply:
Is an primeng directive that will trap the focus of the elements inside, in this case we have a form in the dialog and I want that when the user hit `tab` it only focus the fields of this form and not the ones outside.